docs/user_guide/learn_more.md

```diff
@@ -22,20 +22,23 @@ The following blog posts have previously been published about Venice:
 - 2022: [Open Sourcing Venice – LinkedIn’s Derived Data Platform](https://engineering.linkedin.com/blog/2022/open-sourcing-venice--linkedin-s-derived-data-platform)
 - 2023: [Public CI](https://blog.venicedb.org/public-ci)
 - 2023: [Stable Releases](https://blog.venicedb.org/stable-releases)
+- 2024: [VeniceCon 2024](https://blog.venicedb.org/venicecon-2024)
+- 2024: [Stable Release 0.4.263](https://blog.venicedb.org/stable-release-0-4-263)
 
 Follow us on [HashNode](http://blog.venicedb.org) or [subscribe to the newsletter](https://blog.venicedb.org/newsletter) 
 to hear about it when new posts are published.
 
 ## Talks
 The following talks have been given about Venice:
 
-- 2018: [Venice with Apache Kafka & Samza](https://www.youtube.com/watch?v=Usz8E4S-hZE)
+[//]: # (- 2018: [Venice with Apache Kafka & Samza]&#40;https://www.youtube.com/watch?v=Usz8E4S-hZE&#41;)
 - 2019: [People You May Know: Fast Recommendations over Massive Data](https://www.infoq.com/presentations/recommendation-massive-data/)
 - 2019: [Enabling next generation models for PYMK Scale](https://www.youtube.com/watch?v=znd-Q6IvCqY)
 - 2022: [Open Sourcing Venice](https://www.youtube.com/watch?v=pJeg4V3JgYo)
 - 2023: [What is Derived Data? (and Do You Already Have Any?)](https://www.infoq.com/presentations/derived-data/)
 - 2023: [Partial Updates in Venice](https://www.youtube.com/watch?v=WlfvpZuIa6Q&t=3880s)
-- 2023: [When Only the Last Writer Wins We All Lose: Active-Active Geo-Replication in Venice](https://www.youtube.com/watch?v=yVXtWzAbcmY)
+- 2023: [When Only the Last Writer Wins We All Lose: Active-Active Geo-Replication in Venice](https://www.youtube.com/watch?v=jfbg6IUgVlI)
+- 2024: [The Journey to a Million Ops / Sec / Node in Venice](https://www.infoq.com/presentations/scalable-low-latency/)
```

Comment:
BTW, this happened in 2023.

Reply:
Fixed. I guess the March 16, 2024 in the link was just the date the recording was published.